File: cloud-storage-spring-java/src/main/java/com/github/sevtech/cloud/storage/spring/bean/UploadFileResponse.java
package com.github.sevtech.cloud.storage.spring.bean;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Getter;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@SuperBuilder
@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class UploadFileResponse extends BaseResponse {
	private String fileName;
	private String comment;
}

File: cloud-storage-spring-kotlin/src/main/kotlin/com/github/sevtech/cloud/storage/spring/bean/UploadFileResponse.kt
package com.github.sevtech.cloud.storage.spring.bean

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_EMPTY)
data class UploadFileResponse(
    var fileName: String? = null,
    var status: Int = 0,
    var cause: String? = null,
    var exception: Exception? = null,
    var comment : String? = null
) : BaseResponse(status, cause, exception)